src/js/event-target.js

```diff
@@ -1,12 +1,50 @@
 /**
- * @file event-target.js
+ * @file src/js/event-target.js
  */
 import * as Events from './utils/events.js';
 
+/**
+ * An EventTarget class that seeks to emulate much of the builtin
+ * HTML EventTarget functionalify as well as add all kinds of ease
```

Comment:
Should be "functionalify" and I think the sentence can end there. The rest is kind of vague.